tutorials/tutorial1_sinus-interactive.ipynb

```diff
@@ -4,10 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# DECOMON tutorial #1\n",
-    "\n",
+    " # <span style='background:purple'> <span style='color:White'> DECOMON tutorial #1 </span></span>\n",
+    " \n",
     "_**Bounding the output of a Neural Network trained on a sinusoidal function**_\n",
     "\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"data/decomon.png\" alt=\"Decomon!\" width=\"100\">\n",
+    "</p>\n",
+    "\n",
+    "\n",
+    "\n",
+    "- &#x1F4DA; <a href=\"https://airbus.github.io/decomon\"> <span style='color:purple'>Documentation</span> </a>\n",
+    "- <a href=\"https://github.com/airbus/decomon\"> <span style='color:purple'>Github</span> </a>\n",
+    "- <a href=\"https://github.com/airbus/decomon/blob/master/TUTORIALS.md\"> <span style='color:purple'>More **decomon** tutorials</span> </a>\n",
+    "\n",
+    "\n",
+    "\n",
+    "_Author: [<span style='color:purple'>Melanie DUCOFFE</span>](https://twitter.com/mducoffe)_"
+   ]
+  },
```

Comment:
In the doc, we extract title + description of each notebook tahnks to the first markdown cell. But now the actual description is in the second cell and the first cell contains only links that will be redundant with links already present in the doc. (Moreover the extraction script has trouble to extract the title because of the <span> but this could be probably fixed).
See the doc rendered and in particular the "tutorials" page by downloading the "doc" artifact generated in the github action for this PR.